cloudferrylib/os/actions/attach_used_volumes_via_compute.py

```diff
@@ -28,8 +30,11 @@ def run(self, info, **kwargs):
             if not instance[utl.META_INFO].get(utl.VOLUME_BODY):
                 continue
             for vol in instance[utl.META_INFO][utl.VOLUME_BODY]:
-                if storage_res.get_status(
-                        vol['volume']['id']) != 'in-use':
+                try:
+                    status = storage_res.get_status(vol['volume']['id'])
+                except NotFound:
```

Comment:
This doesn't look right. Each instance volumes must be migrated, if volume is not found in destination cloud - then we somehow missed it in the earlier stages. Please look through.